package.json

```diff
@@ -154,6 +154,16 @@
                         "replace",
                         "allowDuplicates"
                     ]
+                },
+                "numberedBookmarks.decorateWholeLine": {
+                    "type":"boolean",
+                    "default": false,
+                    "description": "Decorate the entire bookmarked line with a background color"
+                },
+                "numberedBookmarks.lineDecoratorColor": {
```

Comment:
Rename the setting to `backgroundLineColor`, with a default value equals to `""`